crates/bevy_render/src/settings.rs

```diff
@@ -93,6 +96,45 @@ impl Default for WgpuSettings {
     }
 }
 
+/// An enum describing how the renderer will initialize resources. This is used when creating the [`RenderPlugin`](crate::RenderPlugin).
+pub enum RenderSettings {
```

Comment:
The manual mode doesn't contain settings. @mockersf suggested calling it `RenderCreation` instead. I much prefer that. With manual creation you provide the handles. With automatic, you provide the settings.

And I think maybe the Default trait can be derived if you annotate the Automatic variant with `#[default]`, right? If not, keep the manual implementation.

Reply:
the `#[default]` attribute can only be used on unit variants